.gitattributes

```diff
@@ -1 +1,5 @@
 * text=auto
+
+# Enforce Windows newlines for C# files to avoid false positives with IDE0055 warning.
+# See https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/issues/106
```

Comment:
```suggestion
# See https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/issues/106.
```

Reply:
You don't need to make the same suggestion on every repo. [If I'm going through with it](https://github.com/Lombiq/NPM-Targets/pull/13#discussion_r879113113) then I'm going to batch process all at once in the command line.

Reply:
Got it.